hw/syn/data/common_syn_cfg.hjson

```diff
@@ -0,0 +1,30 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+{
+  flow:             syn
+  flow_makefile:    "{proj_root}/hw/syn/data/syn.mk"
+
+  import_cfgs:      [// common server configuration for results upload
+                     // TODO: check whether this config file can be aligned such that it can
+                     // be reused among different flow types
+                     // "{proj_root}/hw/dv/data/fusesoc.hjson",
+                     "{proj_root}/hw/data/common_project_cfg.hjson",
+                     "{proj_root}/hw/syn/data/{tool}.hjson"]
+
+  // Default directory structure for the output
+  dut:              "{name}"
+  build_dir:        "{scratch_path}/{build_mode}"
```

Comment:
We can change this later, but ideally, the {build_dir} should be uniquified with {tool} and {dut} so that all downstream paths {directories, logs etc.} can be constants and can be added to this common file.

Reply:
this is derived from the `common_project_cfg.hjson` which does exactly what you explained.